atomix/cluster/src/main/resources/raft-entry-schema.xml

```diff
@@ -67,7 +67,8 @@
 
   <sbe:message name="Meta" id="6">
     <field name="term" id="0" type="uint64"/>
-    <data name="votedFor" id="1" type="varDataEncoding"/>
+    <field name="flushedIndex" id="1" type="uint64"/>
```

Comment:
Nit: Can we rename it? 'flushedIndex' doesn't feel good to my ears. Proposals: lastWrittenIndex, persistedIndex,

Reply:
Sure